pyproject.toml

```
@@ -31,7 +32,7 @@ python = ">=3.9.2,<3.13"
 # Used for local development
 pycountry = "^22.3.5"
 matplotlib = "^3.3.4"
-nautobot = ">=2.4.2,<3.0.0"
+nautobot = "^2.4.16"
```

Please change this to conform to PEP508

```suggestion
nautobot = ">=2.4.16,<3.0.0"
```